package.json

```diff
@@ -59,16 +59,16 @@
     }
   },
   "resolutions": {
-    "@ant-design/cssinjs": "1.0.2",
+    "@ant-design/cssinjs": "1.23.0",
     "enhanced-resolve": "5.10.0"
   },
   "peerDependencies": {
-    "antd": ">=5.8.0",
+    "antd": ">=5.23.0",
```

Comment:
Could you check if you broke support of version 5.8.0? If it's not broken, then we should keep `>=5.8.0`